pgzero/game.py

```diff
@@ -15,16 +15,15 @@
 DISPLAY_FLAGS = 0
 
 
-def exit():
-    """Wait for up to a second for all sounds to play out
-    and then exit
+def exit(exit_status=0):
+    """Cleanly exits pgzero and pygame.
+
+    Args:
+        exit_status (int): Exit status. The default value of 0 indicates
```

Comment:
I don't think we should bother with exit statuses. What you've done makes perfect sense for a general app development framework, but for Pygame Zero I think this adds unnecessary complexity.

We pay a cost for any additional complexity. It makes the documentation longer and adds terms that beginners may not be familiar with (and are unlikely to need). Here I suspect they will not know what an exit status is, and there are very few cases where it would be useful for them.

There is a very slightly stronger case that you can pair up `exit(n)` with `on_exit(n)`, so you can signal different exit conditions. But there are other ways to deal with this, eg. setting global flags.

I guess our expectations differ. I expect `exit()` to be very rarely used, because most games don't need to manage their own quitting. Meanwhile `on_exit()` would be used rarely, eg to save scores or game state.